lib/components/artist/article.js

```diff
@@ -0,0 +1,41 @@
+/* @flow */
+'use strict';
+
+import Relay from 'react-relay';
+import React from 'react-native';
+const { ScrollView, StyleSheet, View, Text } = React;
+
+import colors from '../../../data/colors';
+import SerifText from '../text/serif';
+import ImageView from '../opaque_image_view';
+
+class Article extends React.Component {
+  render() {
+    const article = this.props.article
+    return (
+      <View style={{width: 340}}>
+        <ImageView style={{ height: 200, marginRight: 20 }}
+                     aspectRatio={1.7}
+                     imageURL={article.thumbnail_image.url} />
+        <SerifText style={{margin: 5}} numberOfLines={3}>{article.title}</SerifText>
+        <SerifText style={{margin: 5}}>{article.author.name}</SerifText>
+      </View>
+    );
+  }
+}
+
+export default Relay.createContainer(Article, {
+  fragments: {
+    article: () => Relay.QL`
```

Comment:
@alloy  this triggers an error:  
"RelayContainer: Expected prop `article` supplied to `Article` to be data fetched by Relay. This is likely an error unless you are purposely passing in mock data that conforms to the shape of this component's fragment."

Is my syntax correct here and in the `Articles` component?